[Feature][Resource] Add local resource system support in configuration resource.storage.type=HDFS_LOCAL #9385 (#10265)
apache/dolphinscheduler: docs/docs/en/guide/resource/configuration.md

Configure the file in the following paths: `api-server/conf/common.properties` and `worker-server/conf/common.properties`.

- Change `data.basedir.path` to the local directory path. Please make sure the user who deploy dolphinscheduler have read and write permissions, such as: `data.basedir.path=/tmp/dolphinscheduler`. And the directory you configured will be auto-created if it does not exists.

> - Modify the following two parameters, `resource.storage.type=HDFS` and `fs.defaultFS=file:///` or modify `resource.storage.type=LOCAL` alone, it will use HDFS local mode and will automatically configure `fs.defaultFS=file:///`.

Reviewer: We should suggest users using the `LOCAL` directly, and manually change can be backup
```suggestion
- modify `resource.storage.type=LOCAL` which will use HDFS local mode and will automatically configure `fs.defaultFS=file:///`. And you can also modify the following two parameters, `resource.storage.type=HDFS` and `fs.defaultFS=file:///` manually.
```

Reviewer: Please also change the ZH doc
